myhpi/wagtail_markdown/minutes_extensions.py

```diff
@@ -100,10 +100,10 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.patterns = [
             (
-                r"\|enter\|\(([0-9:]+)\)\(([\w,\ ]+)\)(\((?P<mean_of_participation>.*?)\))?",
+                r"\|enter\|\(([0-9:]+)\)\(([^\)\(]+)\)(\((?P<mean_of_participation>.*?)\))?",
```

Comment:
Like it :) We could even go one step further and support roles like `Dr. Thomas Habbe (Fakultätsmanager)` with this
But I am fine with your solution, too
```suggestion
                r"\|enter\|\(([0-9:]+)\)\(((?:[^\)\(]+(?:\([^\)]+\))?)+)(\((?P<mean_of_participation>.*?)\))?",
```

Reply:
At least on regex101.com this does not work because it does not match the mean of participation.
![image](https://user-images.githubusercontent.com/6863832/171864929-3aee5b6f-1ce7-4640-83fd-5f58ba6be913.png)

Reply:
Ohh, yeah, didn't test it with that. Hang on

Reply:
This should work: \|enter\|\(([0-9:]+)\)\(((?:[^\)\(]+(?:\([^\)]+\))?)+)\)(\((?P<mean_of_participation>.*?)\))?